### DISCUSSION TITLE
If people or looking for bug fixes. This is what I found scattered around.

### DISCUSSION THREAD

**Discussion by ghost** [py-pdf/pypdf#1046] [Show and tell]
If people or looking for bug fixes. This is what I found scattered around, I merged it together: 

https://github.com/Tristan79/PyPDF2/tree/master
## Version 1.26.2t, 2016-06-17
- Write binary data comment (Matthew Stamy, mstamy2)
- When scaling a page, reset the cropBox to match the new mediaBox(Brian Painter, BrianPainter)
- Added URI linking (John Mulligan, JohnMulligan)   
- Fixed another bug with mismatched indirect references (Gurulabs)
- Makes basic_merging.py more pythonic and adds beginner friendly explanation (app-git-hub)
- fix example for python 3 Gustavo Goretkin, goretkin)
- Initial support for CMap character translation (Upliner Mikhalych, Upliner)
## Version 1.26.1t, 2016-05-21
- Added pdf-filter-type-3 (Pekka Pöyry, quantus)
- Try to handle pdf files with invalid stream length (untested eof?) (Pekka Pöyry, quantus)
- NameObject could be encoded with PDFDocEncoding (pavelbrych)
- Support for radio buttons (Shane McQuillan, shanemcquillan)
- Add JPX/DCT & CCITTFax Decode (Sylvain Pelissier, sylvainpelissier)
- Added support for preserving newlines in extractText (JCSchiavone) 
- Improved embedded file handling (Rüdiger Jungbeck, rjungbeck)
  Allow attachment of more than 1 file with PdfFileWriter.addAttachment()
  Allow listing of attachments in PdfFileReader.listAttachments()
  Allow retrival of attachment PdfFileReader.getAttachment()
- When merging pages together, clip by trimBox, which would otherwise be ignored (Forrest Voight, forrestv)
- Use pyCrypto library for faster decryption if available (MarcMann)
- sorted dictionary items before appending to lists (dsk7)
- Allow passing doc permissions (Alfred Hall, ahall)
- MD5 Hash & Bug Fixes (Harry Karvonen, Hatell)
- Prevent infinite loop in readObject() function (Henri Salo, fgeek)

**Comment by mstamy2** [comment]
Thanks a lot for putting all this together. I'm working on some new features myself, and so will need to take some time testing and merging with these.

It's likely we will increment the major version number soon.

**Comment by moodh** [comment]
Any progress on this? The "Allow attachment of more than 1 file with PdfFileWriter.addAttachment()" would solve so many problems for me. :)

**Comment by MartinThoma** [comment]
Is anything of this still open? Would you mind creating PRs for missing features / bug fixes?
